frontend/packages/helm-plugin/src/components/forms/rollback/HelmReleaseRollbackForm.tsx

```diff
@@ -39,8 +39,9 @@ const HelmReleaseRollbackForm: React.FC<Props> = ({
 
   const formHelpText = (
     <Trans t={t} ns="helm-plugin">
-      Select the version to rollback <strong style={{ color: '#000' }}>{{ releaseName }}</strong>{' '}
-      to, from the table below:
+      Select the version to rollback{' '}
```

Comment:
Nit: When using a single string at the beginning we don't need to update the translation.

```suggestion
      {'Select the version to rollback '}
```

Reply:
Updated it as per your suggetion.